src/App.js

```diff
@@ -404,40 +404,45 @@ export function OverallStatus({
   );
 }
 
-export function DisplayCheckResult(
+type DisplayCheckResultProps = {
   title: string,
   actionable: boolean,
-  checkResult: ?CheckResult,
-) {
-  let titleContent = title;
-  if (!actionable) {
-    titleContent = (
-      <div>
-        <Tooltip title="This check is not actionable">
-          <Icon type="notification" /> {title}
-        </Tooltip>
-      </div>
+  checkResult: CheckResult
+}
+class DisplayCheckResult extends React.PureComponent<DisplayCheckResultProps, void> {
```

Comment:
I honestly don't know what that second argument in the `<...>` bracket does. :)

Reply:
Note, I removed the `export` because this component is not needed outside `App.js` as far as I can see.